### PR TITLE
Manual instrumentation bug fixes

### PR DESCRIPTION
**Pull request by pierricgimmig**
- Include parameters when searching for orbit_api functions to hook
  into to  make sure we are hooking the stubs and not the compiled
  implementation of those functions (b/172988899)
- Remove force inlining of non-stub functions as it is not required
  anymore. (Also fixes warning with GCC)
- Include Orbit.h in OrbitTest.cpp, not Tracing.h. (OrbitTest needs to use the api directly)

Tests: 
- Profiling OrbitTest produces expected manual instrumentation scopes
- Profiling OrbitService doesn't generate garbled manual instrumentation scopes

### CODE REVIEW

**Comment by pierricgimmig** [comment]
@ronaldfw , @antonrohr, I'm wondering if you got a notification for the review.  The PR used to be a draft but was converted a few days ago.

File: OrbitClientData/FunctionUtils.cpp [hunk 1]
```diff
@@ -50,13 +50,16 @@ std::unique_ptr<FunctionInfo> CreateFunctionInfo(const SymbolInfo& symbol_info,
   return function_info;
 }
 
+#define STUB_PARAMS \
+  "(unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long)"
```
Comment (antonrohr) [inline OrbitClientData/FunctionUtils.cpp:54]: Cant this be solved without a macro? 
I feel less macros is better than more macros.. but I am unsure

**Review by antonrohr — APPROVED** [review]
Hey @pierricgimmig, sorry that this slipped through the net. Looks good, only one nitpick

File: OrbitClientData/FunctionUtils.cpp (hunk 1 above)
Comment (pierricgimmig) [reply OrbitClientData/FunctionUtils.cpp]: Yeah I agree, although concatenating string literals happen at compile time... I did change it to make the code cleaner.

**Comment by pierricgimmig** [comment]
> Hey @pierricgimmig, sorry that this slipped through the net. Looks good, only one nitpick

No worries, thanks for the review!

**Pull request merged**